scripts/configure-flannel-nodes.sh

```
@@ -0,0 +1,28 @@
+source "$(dirname "${BASH_SOURCE[0]}")/env.sh"
+source "$(dirname "${BASH_SOURCE[0]}")/utils.sh"
+
+# Use hosted cluster kubeconfig if available
+[[ -f "${HOSTED_CLUSTER_NAME}.kubeconfig" ]] && export KUBECONFIG="${HOSTED_CLUSTER_NAME}.kubeconfig"
+
+# Get all used subnets and nodes without CIDR in one pass
+eval $(oc get nodes -o json | jq -r '
+  .items | {
+    used: [.[] | .spec.podCIDR // empty | match("10.244.(\\d+).0").captures[0].string | tonumber] | sort,
+    nodes: [.[] | select(.spec.podCIDR == null) | .metadata.name] | join(" ")
+  } | "used_subnets=(\(.used | @sh)); nodes_to_patch=\"\(.nodes)\""')
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)